modules/gdscript/gdscript_editor.cpp

```diff
@@ -3038,9 +3038,9 @@ ::Error GDScriptLanguage::lookup_code(const String &p_code, const String &p_symb
 			if (!is_function) {
 				// Guess in autoloads as singletons.
 				if (ProjectSettings::get_singleton()->has_autoload(p_symbol)) {
-					const ProjectSettings::AutoloadInfo &singleton = ProjectSettings::get_singleton()->get_autoload(p_symbol);
-					if (singleton.is_singleton) {
-						String script = singleton.path;
+					const ProjectSettings::AutoloadInfo &autoload_singleton = ProjectSettings::get_singleton()->get_autoload(p_symbol);
```

Comment:
I'd name it `autoload` only, whether it's singleton is what the if condition then checks.